nuget/SkiaSharp.Views.Uno.WinUI.nuspec

```diff
@@ -0,0 +1,87 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+
+    <!-- package -->
+    <id>SkiaSharp.Views.Uno.WinUI</id>
+    <title>SkiaSharp for Uno Platform</title>
+    <version>1.0.0</version>
+    <description>
+SkiaSharp for Uno Platform is a set of views that can be used to draw on the screen.
+    </description>
+    <summary>
+SkiaSharp for Uno Platform is a set of views that can be used to draw on the screen.
+    </summary>
+    <releaseNotes>
+Please visit https://go.microsoft.com/fwlink/?linkid=868517 to view the release notes.
+    </releaseNotes>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=868515</projectUrl>
+    <iconUrl>https://go.microsoft.com/fwlink/?linkid=2130524</iconUrl>
+    <tags>ui uno xamarin graphics ios android catalyst windows winui macos cross-platform skiasharp</tags>
+
+    <!-- legal -->
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=868514</licenseUrl>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+
+    <dependencies>
```

Comment:
What should happen if I install this package into a WinUI app? Should we add a net6.0-windows dependency here as well that does nothing or depends on the SkiaSharp.Views.WinUI? It may also need a `_._` file in the windows libs so that nothing gets installed.

Reply:
Indeed, it's useful because of the `netstandard2.0` target, I'll add it back similar to the way it is in the SkiaSharp.Views.Uno.